website/docs/reference/widgets/text.md

```diff
@@ -22,20 +22,22 @@ Additionally, you can use **HTML code** in the Text field to customize the appea
 To display dynamic text that changes based on user interactions or data from external sources, you can use binding values to connect different widgets and their properties.
 
 
-For example, if you have a Table widget with a list of tasks, and you want to display the details of the currently selected task in a Text widget:
+For example, if you have a Table widget that displays a list of tasks, where each row contains the task name, the person assigned to the task, and the deadline for completion.
 
 ```js
-{{Table1.selectedRow.task}}
+Name	| Task	         | Deadline
```

Comment:
here you should be giving the JSON so that the user can use it in table data to check